askama/src/filters/mod.rs

```diff
@@ -511,6 +511,35 @@ pub fn wordcount(s: impl ToString) -> Result<usize, Infallible> {
     wordcount(s.to_string())
 }
 
+/// Returns a titlecased version of the value. I.e. words will start with uppercase letters, all
```

Comment:
Let's avoid "I.e." which is an obscure reference to latin words, and defer the second sentence to a second line of the docstring (after an empty line; and also update the book accordingly).

Also it appears "titlecase" is not an actual English word (per Merriam-Webster and DuckDuckGo), so just use "title case"? (Also in the docs.)

Reply:
Fixed.

Reply:
Note:

> (after an empty line; and also update the book accordingly).

Reply:
:sweat_smile: